src/components/WalletConnect.vue

```diff
@@ -43,9 +43,17 @@ export default defineComponent({
 			walletAddress: "Not Found",
 		};
 	},
+	setup() {
+		const walletStore = WalletStore();
+		return {
+			initializeWebMode: walletStore.setWebMode,
+		};
+	},
```

Comment:
why we have two `walletStore`s? this one looks more reasonable than the one in App.vue (it is using the `setWebMode` rather than `setHasAlgoSigner`

Reply:
walletStore is instance of store which is used for connecting to store from WalletConnect.vue file

Reply:
yes, but why we have two instance? On here and one in App.vue?